tool/compute_tag_analytics.py

```diff
@@ -279,7 +285,16 @@ def _usage_and_exit(error_message):
     with open(NOOK_EXPORT, mode="r") as nook_export_file:
         nook_export_data = json.load(nook_export_file)
     conversation_tags = nook_export_data[CONVERSATION_TAGS_COLLECTION_KEY]
-    nook_conversations = nook_export_data[CONVERSATIONS_COLLECTION_KEY]
+
+    # If the data is nook_conversations, use it, otherwise try to read from nook_conversation_shards
+    if CONVERSATIONS_COLLECTION_KEY in nook_export_data:
+        nook_conversations = nook_export_data[CONVERSATIONS_COLLECTION_KEY]
+    elif CONVERSATIONS_SHARDS_COLLECTION_KEY in nook_export_data:
```

Comment:
I recommend putting this if statement before `if CONVERSATIONS_COLLECTION_KEY` because if there are shards, they should probably take precedence over stale data in `nook_conversations`.

Reply:
That's a good point, thanks, and I see that the UI is also considering shards first, so good idea to do the same here. Fixed now.

Reply:
Ahh, I see this is why the comment was wrong - please update to match new logic

Reply:
thanks for the catch, done!